tasktiger_admin/views.py

```diff
@@ -130,3 +146,11 @@ def queue_detail(self, queue, state):
             n=n,
             tasks=tasks,
         )
+
+
+def convert_keys_to_datetime(dict_arg, keys):
```

Comment:
Is there better place for this fn?

Reply:
I think the proper way to do this would be to have an `Execution` class in TaskTiger and initialize that instead of simply having a dict. Since we don't really use `task.executions` anywhere except to render it in the admin I don't think it would break a lot of things.

However, this might be out of scope for this PR so I'm fine with this workaround for now.

Reply:
Yea that would be nicer. But I would have to find all the possible keys that can be in the executions dict and model the class accordingly. Will skip on that now.